i18n/uk/i18n.json

```diff
@@ -0,0 +1,106 @@
+    "how-it-works-4": "Скопіюйте згенероване вказання ліцензії та вставте його до своєї публікації для правильного використання зображення.",
+    "how-it-works": "Як це працює?",
+    "input-placeholder": "Веб-адреса зображення у Вікісховищі чи статті Вікіпедії.",
+    "go-button": "Розпочнімо!",
+    "feedback-name": "Ім'я",
+    "feedback-body": "Відгук",
+    "feedback-submit": "Надіслати",
+    "feedback-thanks": "Дякуємо за Ваш відгук!",
+    "feedback": "Відгук",
+    "about": "Про",
+    "legal": "Дані та Політика конфіденційності",
+    "legal-tracking-switch": "Зніміть відмітку для припинення відстеження Вашого відвідування.",
+    "legal-hosting": "Цей інструмент розміщено на Вікімедіа Лабс. Прочитайте Умови використання на <a href=\"https://wikitech.wikimedia.org/wiki/Wikitech:Labs_Terms_of_use\" target=\"_blank\">https://wikitech.wikimedia.org/wiki/Wikitech:Labs_Terms_of_use</a>.",
+    "private-use": "Суто приватне використання (без публікації) та інші винятки й обмеження щодо авторських прав"
+  },
+  "error": {
+    "generic": "Сталася непередбачувана помилка. Вибачте.",
+    "mediatype-unsupported": "На жаль, цей тип медіа наразі не підтримується інструментом. Вибачте.",
+    "licence-unsupported": "На жаль, виявлений тип ліцензії наразі не підтримується інструментом. Вибачте.",
+    "url-invalid": "Зазначена Вами веб-адреса не може бути оброблена. Вибачте.",
+    "licence-not-recognized": "На жаль, формат, у якому ліцензійна інформація зберігається для цього зображення, наразі не підтримується інструментом. Вибачте.",
+    "send-feedback": "Щось не спрацювало. Будь ласка, спробуйте знову.",
+    "feedback-generic": "Щось не спрацювало. Будь ласка, спробуйте знову.",
+    "feedback-blank-fields": "Будь ласка, заповніть усі поля форми відгуку."
+  },
+  "info-box": {
+    "private-use": "Для більшості приватних використань Вам не потрібно вказувати ліцензію.",
+    "private-use-more-link": "&#8594; ДІЗНАТИСЯ БІЛЬШЕ",
+    "dont-show-again": "Не показувати це повідомлення знову.",
+    "understood-and-close": "Зрозуміло, закрити.",
+    "ported-licence": "Деякі ліцензії Creative Commons було опубліковано в редакціях для певних країн, так званих «адаптованих ліцензіях». Зображення, для якого Ви запитували вказання ліцензії, опубліковане з такою адаптованою ліцензією. Інструмент не може перевірити, чи ліцензійні вимоги цієї ліцензії Creative Commons для окремої країни еквівалентні англомовній версії. Однак Ви все одно можете згенерувати повідомлення-атрибуцію про ліцензію цим інструментом. Щоб бути повністю певними, ми радимо перевірити вимоги цієї версії ліцензії для країни. Додатково це вказання ліцензії повинне бути мовою її вмісту, щоб виконувати свою функцію повністю."
+  },
+  "dialogue": {
+    "public-domain-picture": "При завантаженні це зображення було позначене як таке, що перебуває у суспільному надбанні, тож Вам не потрібно вказувати ліцензію.",
+    "no-attribution-needed": "Вказання ліцензії не потрібне,
```

Comment:
Missing closing quote here. This makes the whole JSON file not well-formed, and app cannot pick it up.